### DISCUSSION TITLE
Welcome sound from skin doesn't play consistently

### DISCUSSION THREAD

**Discussion by SevenChords** [ppy/osu#22465] [Q&A] [answered]
When i start the game, the welcome.wav file from the skin will only play sometimes, not all of the time (same skin, intro sequence always set to welcome). sometimes instead of the skins welcome.wav the default welcome to osu voice plays.
haven't been able to spot any pattern in when it plays or doesn't play.
edit: same goes for the seeya.wav, sometimes it plays, sometimes the default voice plays

**Comment by ItsJustSomeDude** [comment]
Please upload the skin if possible.

**Reply by SevenChords** [reply]
whole skin is too big to upload here, but i can give you the welcome and seeya soundfiles, since those are what is problematic (had to zip them cause wav isn't supported on github)
[welcome.wav.zip](https://github.com/ppy/osu/files/10557913/welcome.wav.zip)
[seeya.wav.zip](https://github.com/ppy/osu/files/10557912/seeya.wav.zip)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/22492
